spec/controllers/games_controller_spec.rb

```diff
@@ -136,4 +136,186 @@
       expect(response).to redirect_to(games_url)
     end
   end
+
+  #Adds white pieces to board
+
+  describe "populate board" do
+    it 'Adds correct number of white Pawns to the board' do
+     game = Game.create!
+     game.populate_board!
+
+     expect(game.pieces.where(type: 'Pawn', color: 'White')).to eq (game.pieces.where(position_x: 6, position_y: 0))
+     expect(game.pieces.where(type: 'Pawn', color: 'White')).to eq (game.pieces.where(position_x: 6, position_y: 1))
+     expect(game.pieces.where(type: 'Pawn', color: 'White')).to eq (game.pieces.where(position_x: 6, position_y: 2))
+     expect(game.pieces.where(type: 'Pawn', color: 'White')).to eq (game.pieces.where(position_x: 6, position_y: 3))
+     expect(game.pieces.where(type: 'Pawn', color: 'White')).to eq (game.pieces.where(position_x: 6, position_y: 4))
+     expect(game.pieces.where(type: 'Pawn', color: 'White')).to eq (game.pieces.where(position_x: 6, position_y: 5))
+     expect(game.pieces.where(type: 'Pawn', color: 'White')).to eq (game.pieces.where(position_x: 6, position_y: 6))
+     expect(game.pieces.where(type: 'Pawn', color: 'White')).to eq (game.pieces.where(position_x: 6, position_y: 7))
+  end
+end
+
+describe "populate board" do
+  it 'Adds white Rook to the board' do
+    game = Game.create!
+    game.populate_board!
+
+expect(game.pieces.where(type: 'Rook', color: 'White')).to eq (game.pieces.where(position_x: 7, position_y: 0))
+end
+end
```

Comment:
take a look at these specs ^ (and below)

Reply:
Remember to look through your files changed tab on PRs. Scan it for things that look off.

Reply:
Look at travis. See the rubocop errors.